package.json

```diff
@@ -28,7 +28,7 @@
     "expo-secure-store": "~9.3.0",
     "expo-status-bar": "~1.0.3",
     "expo-web-browser": "~8.6.0",
-    "firebase": "^8.4.1",
+    "firebase": "^7.9.0",
```

Comment:
why the downgrade? because of the expo warning? I've had issues with this in the past whereby an older version of the firebase package wouldn't work, so I accepted the expo warning for the sake of having an up-to-date firebase version

Reply:
Yes because of warning. And at that time my goal was to get it up and running without warnings/issues to do some development. Now I'll focus on upgrading the SDKs.